client/components/Editor/index.jsx

```diff
@@ -145,6 +202,14 @@ class Editor extends Component {
     }
 
     render() {
+        const scenarioStatusMenuItem = this.props.status !== undefined && (
```

Comment:
I like how status is being passed into this component. Status, being determined in setScenario